Sources/SQL/Data/DataSubqueryColumn.swift

```diff
@@ -0,0 +1,17 @@
+/// A computed SQL column.
+public struct DataSubqueryColumn {
+    /// The SQL sub-query
+    public var query: String
```

Comment:
Subquery should be a query struct, not just a string. The serializer should take care of serializing it.